Octokit.Tests.Integration/Clients/RepositoryCollaboratorClientTests.cs

```diff
@@ -136,4 +136,26 @@ public class TheIsCollaboratorMethod
             }
         }
     }
+
+    public class TheInviteMethod
+    {
+        [IntegrationTest]
+        public async Task CanInviteNewCollaborator()
+        {
+            var github = Helper.GetAuthenticatedClient();
+            var repoName = Helper.MakeNameWithTimestamp("public-repo");
+
+            using (var context = await github.CreateRepositoryContext(new NewRepository(repoName)))
+            {
+                var fixture = github.Repository.Collaborator;
+                var permission = new CollaboratorRequest();
+
+                // invite a collaborator
+                var response = await fixture.Invite(context.RepositoryOwner, context.RepositoryName, "maddin2016", permission);
```

Comment:
>  you cannot delete a repo until there is a pending invitation
Did you mean cannot delete *when* there is a pending invite?  If so then I guess you could `Delete` the invitation before trying to delete the repo?

Im not sure about why the email is not received.  I will try some invites myself when I can...